docs/syntax_guide.rst

```diff
@@ -266,13 +279,13 @@ Operators
 .. list-table::
    :header-rows: 1
 
-   * - Heading Operators
+   * - Orientation Operators
      - Meaning
    * - :sampref:`{scalar} deg`
-     - The given heading, interpreted as being in degrees
+     - The given angle, interpreted as being in degrees
    * - :sampref:`{vectorField} at {vector}`
-     - The heading specified by the vector field at the given position
-   * - :sampref:`{direction} relative to {direction}`
+     - The orientation specified by the vector field at the given position
+   * - :sampref:`({heading} | {vectorField}) relative to ({heading} | {vectorField})`
```

Comment:
Will need to include `Orientation` here; it's probably more readable to keep it as `direction` in the code snippet and then have the text start "The first direction (a heading, orientation, or vector field)...".

Reply:
I meant the syntax line needs to contain `Orientation` as one of the 3 options in the unions, unless you change them back to just `direction` (which I think would be better).

Reply:
Ah good point. I'll swap it back to direction.